python-frontend/src/main/java/org/sonar/python/types/RuntimeType.java

```diff
@@ -31,6 +31,9 @@
 
 public class RuntimeType implements InferredType {
 
+  // Name of the type returned by the type() function
+  private static final String TYPE_CLASS_NAME = "type";
```

Comment:
Did you try to use `org.sonar.python.types.InferredTypes#TYPE`?
It looks like that for the v1 type inference we have this constant representing the `type` class.

Reply:
good catch, I ll use it. thanks